cfme/tests/services/test_generic_service_catalogs.py

```diff
@@ -26,10 +26,10 @@
 def dialog():
     dialog = "dialog_" + fauxfactory.gen_alphanumeric()
     element_data = dict(
+        choose_type="Text Box",
```

Comment:
This movement, is it intended? If so, we should investigate why this happens, because the order of a dict is not guaranteed, so we should not rely on it.